src/api/production-api-client.ts

```diff
@@ -21,7 +27,13 @@ class ProductionApiClient implements ApiClient, LoggerApiClient {
 		const response = await fetch( request );
 
 		if ( response.ok ) {
-			return response.json();
+			const data = await response.json();
+
+			const cacheExpiryTime = Date.now() + 7 * 24 * 60 * 60 * 1000; // 7 days in milliseconds
```

Comment:
**Question (non-blocking)**: As I've been thinking about this, I think I've had a small change of heart -- what do you think if we start with three days? I think I want to start more cautiously, especially as the start will be when there are the most updates to the reporting config. Does that seem like a good balance to you? And sorry for the back and forth! 😆